q2_types/feature_table/_type.py

```diff
@@ -30,5 +30,10 @@
 
 plugin.register_semantic_type_to_format(
```

Comment:
Right now we only support registering on artifact format on a semantic type. As I understand it, we just want to support the new V210 format. Ultimately we only need one `register_semantic_type_to_format` here.